### PR TITLE
Added set- and getUserRolesById endpoints to UserController

### PR DESCRIPTION
**Pull request by Igor-Polatajko**

### CODE REVIEW

File: src/main/java/com/smartcity/controller/UserController.java [hunk 1]
```diff
@@ -61,4 +60,15 @@ public boolean updatePassword(@RequestBody String newPassword,
         return userService.updatePassword(userId, newPassword);
     }
 
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/{id}/get-roles", produces = MediaType.APPLICATION_JSON_VALUE)
+    List<Role> getRolesByUserId(@PathVariable("id") Long id) {
```
Comment (iholovatyi) [inline src/main/java/com/smartcity/controller/UserController.java]: Add test for this method on Service level

File: src/main/java/com/smartcity/controller/UserController.java [hunk 2]
```diff
@@ -61,4 +60,15 @@ public boolean updatePassword(@RequestBody String newPassword,
...
+        return userService.getRoles(id);
+    }
+
+    @ResponseStatus(HttpStatus.OK)
+    @PutMapping(value = "/{id}/set-roles", consumes = MediaType.APPLICATION_JSON_VALUE)
+    boolean setRolesUserId(@PathVariable("id") Long userId, @RequestBody List<Role> roles) {
```
Comment (iholovatyi) [inline src/main/java/com/smartcity/controller/UserController.java]: Add test for this method on Service level

File: src/main/java/com/smartcity/controller/UserController.java (hunk 1 above)
Comment (iholovatyi) [reply src/main/java/com/smartcity/controller/UserController.java]: Is that default one?

File: src/main/java/com/smartcity/dao/RoleDaoImpl.java [hunk 3]
```diff
@@ -48,7 +48,8 @@ public Role create(Role role) {
             role.setId(Optional.ofNullable(holder.getKey().longValue())
                     .orElseThrow(() -> new DbOperationException("Create Role Dao method error: AutoGeneratedKey = null")));
             return role;
-        } catch (Exception e) {
+        }
```
[the remark this answers is not in the document]
Comment (iholovatyi) [reply src/main/java/com/smartcity/dao/RoleDaoImpl.java]: Is that default one?

[the remark this answers is not in the document]
Comment (Igor-Polatajko) [reply src/main/java/com/smartcity/dao/RoleDaoImpl.java]: No, it's custom)

**Pull request merged**